packages/rancher-monitoring/package.yaml

```diff
@@ -1,4 +1,7 @@
-url: https://kubernetes-charts.storage.googleapis.com/prometheus-operator-8.16.1.tgz
+url: https://github.com/prometheus-community/helm-charts.git
+subdirectory: charts/kube-prometheus-stack
+type: git
+commit: 5c8e31ae6cb6e355900e2cbbd5cbceaec0c85821
```

Comment:
There a reason we don't want to use their index.yaml ? https://prometheus-community.github.io/helm-charts/index.yaml
I've seen some repos that do text manipulation between the tree and publishing to the index.

Reply:
No particular reason. We can switch to using the index in the next commit.